stock_available/views/product_product_view.xml

```diff
@@ -0,0 +1,71 @@
+<!-- Copyright 2014 Numérigraphe
+     Copyright 2016 Sodexis
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html). -->
+<odoo>
+    <record model="ir.ui.view" id="product_normal_form_view">
+        <field name="name">Quantity available to promise (variant form)</field>
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="stock.product_form_view_procurement_button" />
+        <field name="arch" type="xml">
+            <xpath
+                expr="//button[@name='action_product_forecast_report']"
+                position="after"
+            >
+                <button
+                    type="object"
+                    name="action_open_quants"
+                    invisible="type != 'consu'"
+                    class="oe_stat_button"
+                    icon="fa-building-o"
+                >
+                    <div class="o_field_widget o_stat_info">
+                        <span class="o_stat_value d-flex gap-1">
+                            <field
+                                name="immediately_usable_qty"
+                                widget="statinfo"
+                                nolabel="1"
+                                class="oe_inline"
+                            />
+                            <field name="uom_name" class="oe_inline" />
+                        </span>
+                        <span class="o_stat_text">Available</span>
+                    </div>
+                </button>
+                <button
+                    type="object"
+                    name="action_open_quants"
+                    invisible="type != 'consu'"
+                    class="oe_stat_button"
+                    icon="fa-building-o"
+                >
+                    <div class="o_field_widget o_stat_info">
+                        <span class="o_stat_value d-flex gap-1">
+                            <field
+                                name="potential_qty"
+                                widget="statinfo"
+                                nolabel="1"
+                                class="oe_inline"
+                            />
+                            <field name="uom_name" class="oe_inline" />
+                        </span>
+                        <span class="o_stat_text">Potential</span>
+                    </div>
+                </button>
+            </xpath>
+        </field>
+    </record>
+    <record model="ir.ui.view" id="view_stock_available_product_list">
+        <field name="name">Quantity available to promise (variant list)</field>
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="stock.view_stock_product_tree" />
+        <field name="arch" type="xml">
+            <field name="virtual_available" position="after">
+                <field
+                    name="immediately_usable_qty"
+                    decoration-danger="immediately_usable_qty &lt; 0"
```

Comment:
Analogous to `virtual_available`, set `optional="show"` and `groups="stock.group_stock_user"` (ref: https://github.com/odoo/odoo/blob/686628f55f6ec/addons/stock/views/product_views.xml#L46)